covid19/utils.py

```diff
@@ -104,3 +104,11 @@ def is_valid_end_date(end_date: str) -> bool:
     end_date_formatted = datetime.strptime(end_date, DATETIME_FORMAT)
 
     return end_date_formatted <= yesterday
+
+
+def get_iso_date(date):
```

Comment:
this is very nicely written

Reply:
That little helper function might be one of my favorite in this project.